expressEngine/model/collection/product.js

```diff
@@ -30,15 +30,18 @@ const productsSchema = new Schema({
     required: false,
   },
 
-}, { collection: 'product', timestamps: true });
+}, {
+    collection: 'product',
+    timestamp: true,
+  });
 
 productsSchema.pre('save', function preSave() {
   const product = this;
   product.id = product._id.toString();
 });
 
+const Product = model('Products', productsSchema);
```

Comment:
use mongoose.model, using only model shoes not defined error

Reply:
i have import model from mongoose.

Reply:
done